Libplanet/FixedSizedQueue.cs

```diff
@@ -0,0 +1,55 @@
+using System.Collections.Concurrent;
+
+namespace Libplanet
+{
+    /// <summary>
+    /// Implements a <see cref="ConcurrentQueue{T}"/> of a fixed size.
+    /// </summary>
+    /// <typeparam name="T">Specifies the type of elements in the queue.</typeparam>
+#pragma warning disable MEN002
+    // This class referenced the implementation of the following.
+    // https://github.com/couchcoding/Logbert/blob/bbed28fabf9493310f4b51324f8af88c14551c9d/src/Logbert/Helper/FixedSizedQueue.cs
```

Comment:
How about shortening the URL like this and removing suppression warning?

```suggestion
    // https://git.io/Jvs1f
```